deploy.sh

```diff
@@ -11,11 +11,13 @@ if [ "$1" = "prod" ]; then
   read answer
   if [ "$answer" != "${answer#[Yy]}" ] ;then
     scp -r dist/* $2@athena.dialup.mit.edu:/mit/courseroad/web_scripts/courseroad/
+    scp -r .htaccess $2@athena.dialup.mit.edu:/mit/courseroad/web_scripts/courseroad/
```

Comment:
oof this is going to take twice as many logins now, is there a way to scp two things at once?

Reply:
We can `scp` two things at once provided that they're headed for the same location. This would resolve it for the `production` case but not the `development` case though since that needs to be copied as `.htaccess` (requiring a rename and counting as a separate "destination").

Reply:
(resolved by naming the dev one .htaccess and putting it in a folder)